yt_dlp/extractor/adobepass.py

```diff
@@ -1724,10 +1801,11 @@ def extract_redirect_url(html, url=None, fatal=False):
                 try:
                     session = self._download_webpage(
                         self._SERVICE_PROVIDER_TEMPLATE % 'session', video_id,
-                        'Retrieving Session', data=urlencode_postdata({
+                        'Retrieving Session', data=urlencode_postdata(filter_dict({
                             '_method': 'GET',
                             'requestor_id': requestor_id,
-                        }), headers=mvpd_headers)
+                            'reg_code': reg_code,
+                        })), headers=mvpd_headers)
                 except ExtractorError as e:
                     if not mso_id and isinstance(e.cause, HTTPError) and e.cause.status == 401:
```

Comment:
Not for this PR, but there should be a `utils.HTTPError_or_none(maybeExtractorErrorFromHTTPError, *withHTTPcodes)`, or `networking....` if you will, for this common usage.